### ISSUE TITLE
No module named 'Cython'

### ISSUE THREAD

**Issue by 1kastner** [bsmurphy/PyKrige#70]
Hi there, first of all thanks for the project!

Somehow I struggle to install it when using Windows + Miniconda.

    (C:\path\to\miniconda)` C:\current\directory>pip install pykrige
    Collecting pykrige
      Downloading PyKrige-1.3.0.tar.gz (596kB)
        100% |████████████████████████████████| 604kB 1.5MB/s
        Complete output from command python setup.py egg_info:
            Traceback (most recent call last):
              File "<string>", line 1, in <module>
              File "C:\Users\current-user\AppData\Local\Temp\pip-build-phddipzi\pykrige\setup.py", line 14, in <module>
            from Cython.Distutils import build_ext
        ModuleNotFoundError: No module named 'Cython'

        ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in C:\Users\current-user\AppData\Local\Temp\pip-build-phddipzi\pykrige\

**Comment by 1kastner** [comment]
Now I have manually installed Cython by typing

    pip install Cython

and it worked. It looks like it only needs to be added to the dependencies?

**Comment by rth** [comment]
@1kastner Hmm, cython is an optional dependency, the installation should have raised a warning instead of failing if it's not installed.  Thanks for reporting it, sounds like a bug. What is the output of `pip --version` on your machine?

We should also probably submit PyKrige to [conda forge](https://conda-forge.org/) so it's easier to install with conda...

**Comment by rth** [comment]
I double checked, that's indeed an issue with `1.3.0` on PyPi. This was fixed since in the master branch, and the fix will be available on PyPi once we release 1.4.0 (cf #67 ). Meanwhile, a temporary fix would be to install from Github,
```
pip install git+https://github.com/bsmurphy/PyKrige.git@master
```

**Comment by 1kastner** [comment]
ok thank you for checking this!

**Comment by bsmurphy** [comment]
Thanks for checking on this, @rth. I will either load v 1.4.0 onto PyPi or get you the credentials in the next few days... (I'm aiming by Monday. Really need to just clear out a solid chunk of time to work on all this stuff...)

**Comment by rth** [comment]
Fixed with version 1.3.1 on PyPi. Please reopen if needed.
